### PR TITLE
🐛 amp-ad doubleclick: Fix fluid impression not being fired in shadow doc

### PR DESCRIPTION
**Pull request by zombifier**
The element was being appended to the real document instead of the ampdoc.

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -1628,16 +1628,18 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
...
-              'src': url,
-              'referrerpolicy': scrubReferer ? 'no-referrer' : '',
-            })
-          )
-        );
+        this.getAmpDoc()
+          .getBody()
+          .appendChild(
+            createElementWithAttributes(
+              this.win.document,
```
Comment (jeffkaufman) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:1635]: Is this.win.document still correct?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]
Can there be tests?

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js (hunk 1 above)
Comment (zombifier) [reply extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js:1635]: Yes (see a similar usage above). The param is to get an object to call `createElement` on.

**Comment by zombifier** [comment]
Added a shadowdoc configuration to the general test suite. Can confirm they fail with the code change reverted. There's no test changes otherwise but the extra indents caused a lot of reformatting making the line count rather deceptive.

**Review by jeffkaufman — APPROVED** [review]

**Pull request merged**